diff -rupN krpc/src/kRPC/Schema/KRPC.cs krpc-new/src/kRPC/Schema/KRPC.cs
--- krpc/src/kRPC/Schema/KRPC.cs	2014-02-20 12:28:44.666910121 +0000
+++ krpc-new/src/kRPC/Schema/KRPC.cs	2014-02-20 12:32:32.258920005 +0000
@@ -861,7 +861,7 @@ namespace KRPC.Schema.KRPC {
     
     public override bool IsInitialized {
       get {
-        foreach (global::KRPC.Schema.KRPC.Service element in ServicesList) {
+        foreach (global::KRPC.Schema.KRPC.Service element in Services_List) {
           if (!element.IsInitialized) return false;
         }
         return true;
